bosh-stemcell/lib/bosh/stemcell/operating_system.rb

```diff
@@ -23,7 +24,8 @@ def ==(other)
 
     class Ubuntu < Base
       def initialize(version)
-        super(name: 'ubuntu', version: version)
+        (version, variant) = version.split('-') unless version.nil?
```

Comment:
Another double negative. This one my brain has an easier time parsing, but `if version` still might be better?